CHANGELOG.md

```diff
@@ -8,21 +8,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Implement SQL source for primary data [#34](https://github.com/ie3-institute/simona/issues/34), [#101](https://github.com/ie3-institute/simona/issues/101)
+- Relevant scientific papers have been added to the documentation [#139](https://github.com/ie3-institute/simona/issues/139)
+- Add troubleshooting section to Users guide [#160](https://github.com/ie3-institute/simona/issues/160)
+- Adapt documentation to changed simonaAPI [#191](https://github.com/ie3-institute/simona/issues/191)
+- Implementing a new plugin framework for external simulations [#195](https://github.com/ie3-institute/simona/issues/195)
```

Comment:
Honestly, these belong more to the `Changed` section

Reply:
Thanks a lot for all your comments!